README.md

```diff
@@ -48,11 +51,223 @@ Creating network "client_default" with the default driver
 Creating cpa-web      ... done
 ```
 
+### Running with Docker ###
+
+To run the `cisagov/example` image via Docker:
+
+```console
+docker run cisagov/example:0.0.1
+```
```

Comment:
This section and all newly-added sections below it should be customized for this project or updated with existing content.